java/client/src/main/java/glide/api/models/commands/GeoAddOptions.java

```diff
@@ -0,0 +1,23 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.GeospatialIndicesBaseCommands;
+import java.util.Map;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * Optional arguments for {@link GeospatialIndicesBaseCommands#geoadd(String, Map, GeoAddOptions)}
+ * command.
+ *
+ * @see <a href="https://redis.io/commands/geoadd/">redis.io</a>
+ */
+@Getter
```

Comment:
I propose to use constructor (or 2) instead of builder - it is easier for an user

Reply:
Sure, but I feel like we have builders like this in the repo so could be wort going over them.

Reply:
I'm planning to replace builders for some existing options by contsructors, for example
```java
InfoOptions.builder().section(SERVER).section(CLUSTER).build();
```
and
```java
new InfoOptions(SERVER, CLUSTER);
```

Reply:
Agreed makes sense.